docs/ContainerCustomization.md

```diff
@@ -103,23 +102,91 @@ ContainerPort items have two properties:
 This item adds a metadata label to the container. Labels have no impact on the container at runtime, but are often used to store version and authoring metadata for use by security scanners and other infrastructure tools.
 
 ContainerLabel items have two properties:
+
 * Include
   * The key of the label
 * Value
   * The value of the label - this may be empty
 
+See [default container labels](#default-container-labels) for a list of labels that are created by default.
+
 ```xml
 <ItemGroup>
     <ContainerLabel Include="org.contoso.businessunit" Value="contoso-university" />
 <ItemGroup>
 ```
 
-## Unsupported properties
+## ContainerEnvironmentVariable
+
+This item adds a new environment variable to the container. Environment variables will be accessible to the application running in the container immediately, and are often used to change the runtime behavior of the running application.
+
+ContainerEnvironmentVariable items have two properties:
```

Comment:
The use of the word "properties" here I'd like to reconsider at some future time, to disambiguate from MSBuild properties. But fine for now, especially since it's preexisting.

Reply:
I was going to use `metadata` pervasively, but I wasn't sure how widely-known that term is among more novice readers, and I didn't want to use `attributes` because strictly speaking they're not attributes (though metadata can be expressed as XML attributes).